diff --git a/node_modules/tinacms/dist/index.js b/node_modules/tinacms/dist/index.js
index 44a5a18..30e70ef 100644
--- a/node_modules/tinacms/dist/index.js
+++ b/node_modules/tinacms/dist/index.js
@@ -45,7 +45,9 @@ import * as DropdownMenuPrimitive from "@radix-ui/react-dropdown-menu";
 import * as ToolbarPrimitive from "@radix-ui/react-toolbar";
 import * as SeparatorPrimitive from "@radix-ui/react-separator";
 import * as TooltipPrimitive from "@radix-ui/react-tooltip";
-import colorString, { get as get$1, to } from "color-string";
+import colorStringPkg from "color-string";
+const colorString = colorStringPkg.default ?? colorStringPkg;
+const { get: get$1, to } = colorString;
 import { ElementApi, isType, NodeApi, mergeProps, createSlatePlugin } from "@udecode/plate";
 import { createForm, FORM_ERROR, getIn } from "final-form";
 import arrayMutators from "final-form-arrays";
@@ -79,7 +81,7 @@ import { IndentListPlugin } from "@udecode/plate-indent-list/react";
 import { NodeIdPlugin } from "@udecode/plate-node-id";
 import { offset, flip, useVirtualFloating, getSelectionBoundingClientRect } from "@udecode/plate-floating";
 import { getLinkAttributes } from "@udecode/plate-link";
-import "@heroicons/react/solid";
+import "@heroicons/react/solid/index.js";
 import { autoformatSmartQuotes, autoformatPunctuation, autoformatLegal, autoformatArrow, autoformatMath } from "@udecode/plate-autoformat";
 import * as yup from "yup";
 import moment from "moment";
